src/Entities/LeashKnot.h

```diff
@@ -0,0 +1,45 @@
+
+#pragma once
+
+#include "HangingEntity.h"
+
+
+
+
+
+// tolua_begin
+class cLeashKnot :
+	public cHangingEntity
+{
+	typedef cHangingEntity super;
+
+public:
+
+	// tolua_end
+
+	CLASS_PROTODEF(cLeashKnot)
+
+	cLeashKnot(eBlockFace a_BlockFace, double a_X, double a_Y, double a_Z);
+
+	/** Looks for mobs leashed to a player and ties them to this knot */
+	void TieNearbyMobs(cPlayer & a_Player, bool a_ShouldBroadCast);
```

Comment:
The name is somewhat misleading, it doesn't tie all nearby mobs. `TiePlayersMobs` might be better.